package.json

```diff
@@ -14,143 +14,140 @@
+    "@typescript-eslint/parser": "^3.7.0",
+    "autoprefixer": "^9.8.5",
+    "babel-jest": "^26.1.0",
+    "babel-loader": "^8.1.0",
     "clean-webpack-plugin": "^3.0.0",
     "core-js": "^3.6.5",
     "coveralls": "^3.1.0",
-    "css-loader": "^3.5.3",
-    "eslint": "^7.0.0",
+    "css-loader": "^3.6.0",
+    "eslint": "^7.5.0",
     "eslint-config-prettier": "^6.11.0",
-    "eslint-import-resolver-webpack": "^0.12.1",
-    "eslint-plugin-flowtype": "^5.1.0",
-    "eslint-plugin-flowtype-errors": "^4.1.0",
-    "eslint-plugin-import": "^2.20.1",
-    "eslint-plugin-prettier": "^3.1.3",
-    "eslint-plugin-react": "^7.18.3",
-    "fetch-mock": "^9.4.0",
+    "eslint-import-resolver-typescript": "^1.1.1",
+    "eslint-import-resolver-webpack": "^0.12.2",
+    "eslint-plugin-import": "^2.22.0",
+    "eslint-plugin-jest": "^23.18.0",
+    "eslint-plugin-prettier": "^3.1.4",
+    "eslint-plugin-react": "7.20.0",
+    "eslint-plugin-react-hooks": "^4.0.8",
+    "eslint-plugin-simple-import-sort": "^5.0.3",
+    "fetch-mock": "^9.10.4",
     "file-loader": "^6.0.0",
-    "flow-bin": "^0.129.0",
-    "flow-typed": "^3.1.0",
     "html-webpack-plugin": "^4.3.0",
     "husky": "^4.2.5",
-    "i18next-scanner-webpack": "^0.5.0",
-    "jest": "^26.0.1",
+    "i18next-scanner-webpack": "^0.5.1",
+    "jest": "^26.1.0",
     "jest-junit": "^11.0.1",
-    "lint-staged": "^10.1.7",
     "mini-css-extract-plugin": "^0.9.0",
     "node-fetch": "^2.6.0",
     "npm-run-all": "^4.1.5",
-    "nyc": "^15.0.1",
+    "nyc": "^15.1.0",
     "optimize-css-assets-webpack-plugin": "^5.0.3",
     "postcss-loader": "^3.0.0",
     "prettier": "^2.0.5",
+    "react-test-renderer": "^16.13.1",
     "redux-mock-store": "^1.5.4",
-    "sass": "^1.26.5",
+    "sass": "^1.26.10",
     "sass-loader": "^9.0.2",
-    "stylelint": "^13.3.3",
-    "stylelint-config-prettier": "^8.0.1",
+    "stylelint": "^13.6.1",
+    "stylelint-config-prettier": "^8.0.2",
     "stylelint-config-recommended": "^3.0.0",
-    "stylelint-prettier": "^1.1.1",
-    "stylelint-scss": "^3.17.1",
+    "stylelint-prettier": "^1.1.2",
+    "stylelint-scss": "^3.18.0",
+    "svg-inline-loader": "^0.8.2",
+    "typescript": "^3.9.7",
     "url-loader": "^4.1.0",
     "webpack": "^4.43.0",
-    "webpack-cli": "^3.3.11",
-    "webpack-dev-server": "^3.10.3",
+    "webpack-cli": "^3.3.12",
+    "webpack-dev-server": "^3.11.0",
     "webpack-merge": "^5.0.9"
   },
   "husky": {
     "hooks": {
-      "pre-commit": "lint-staged",
       "prepare-commit-msg": "./bin/prepare-commit-msg"
     }
-  },
-  "lint-staged": {
-    "*.js": [
-      "prettier --write",
-      "eslint --plugin flowtype-errors --rule 'flowtype-errors/show-errors: 2' --rule 'flowtype-errors/show-warnings: 1'"
-    ],
-    "*.scss": [
-      "prettier --write",
-      "stylelint"
-    ],
-    "*.py": [
-      "isort",
-      "black",
-      "flake8"
-    ],
-    "*.{json,yml}": [
-      "prettier --write"
-    ]
```

Comment:
Are these replaced by something else?

Reply:
Hm, they are not currently -- other than CI, which will catch them.

Reply:
(That's the pattern we've been using on other projects, where we can't ensure users have the dependencies installed locally.)

Reply:
I find them very helpful and would like to keep them (maybe it can be done in a way that silently passes if the dependencies aren't installed?)

Reply:
(That doesn't have to hold up merging this PR though.)

Reply:
Yeah, I'm happy to consider that, if we can find a way for it to silently pass if the deps are missing.